package.json

```diff
@@ -26,7 +26,7 @@
     "del": "^1.1.1",
     "eslint-plugin-react": "^2.0.1",
     "eventemitter3": "^1.0.1",
-    "express": "^4.11.2",
```

Comment:
Did you realy removed express.js from packages? You are using it in `server/express.js`